tests/integration_tests.py

```diff
@@ -0,0 +1,174 @@
+    response = requests.request("POST", base_url + endpoint, headers=headers,
+                                data=payload_objects)
+    object_id = response.text.replace("\"", "").replace("\n", "")
+    assert response.status_code == 200
+
+
+def test_get_objects():
+    """Test `GET /objects/{object_id}` for fetching info about DrsObject."""
+    global access_id
+    is_expand = "True"
+    endpoint = "/objects/" + object_id + "?expand=" + is_expand
+    response = requests.request("GET", base_url + endpoint, headers=headers)
+    response_content = json.loads(response.text)
+    access_id = response_content['access_methods'][0]['access_id']
+    assert response.status_code == 200
+    assert response_content['name'] == json.loads(payload_objects)['name']
+
+
+def test_get_objects_access():
+    """Test `GET /objects/{object_id}/access/{access_id}` for fetching URL to
+    fetch the bytes of a DrsObject
+    """
+    endpoint = "/objects/" + object_id + "/access/" + access_id
+    response = requests.request("GET", base_url + endpoint, headers=headers)
+    response_content = json.loads(response.text)
+    assert response.status_code == 200
+    assert response_content['url'] == \
+        json.loads(payload_objects)['access_methods'][0]['access_url']['url']
+
+
+def test_put_objects():
+    """Test `PUT /objects/{object_id}` to modify or update a DrsObject"""
+    global access_id
+    endpoint = "/objects/" + object_id
+    object_put_payload = json.loads(payload_objects)
+    object_put_payload['name'] = "gerp_constrained_elements.tetraodon_" + \
+        "nigroviridis.bed.gz"
+    response = requests.request("PUT", base_url + endpoint, headers=headers,
+                                data=json.dumps(object_put_payload))
+    assert response.status_code == 200
+    is_expand = "True"
+    endpoint = "/objects/" + object_id + "?expand=" + is_expand
+    response = requests.request("GET", base_url + endpoint, headers=headers)
+    response_content = json.loads(response.text)
+    access_id = response_content['access_methods'][0]['access_id']
+    assert response.status_code == 200
+    assert response_content['name'] == \
+        "gerp_constrained_elements.tetraodon_nigroviridis.bed.gz"
+
+
+def test_delete_objects_access():
+    """Test `DELETE /objects/{object_id}/access/{access_id}` to delete
+    DrsObject's access method.
+    """
+    endpoint = "/objects/" + object_id + "/access/" + access_id
+    response = requests.request("DELETE", base_url + endpoint, headers=headers)
+    assert response.status_code == 200
+    assert response.text.replace("\"", "").replace("\n", "") == access_id
+
+
+def test_delete_objects():
+    """Test `DELETE /objects/{object_id}` to delete existing DrsObject"""
+    endpoint = "/objects/" + object_id
+    response = requests.request("DELETE", base_url + endpoint, headers=headers)
+    assert response.status_code == 200
+    assert response.text.replace("\"", "").replace("\n", "") == object_id
```

Comment:
Would be good to add test cases for some errors as well, especially 404 and 400. Eventually we should cover 401 and 403 as well, but we can do that once we have a better idea of how controlled access management will look like